### ISSUE TITLE
customize input class

### ISSUE THREAD

**Issue by gyamxxx** [vue-bulma/datepicker#8]
Hi, I'm sorry poor English.
could you add :class attribute in these files for customizing class ?

https://github.com/vue-bulma/datepicker/blob/master/src/SingleInput.vue#L2
https://github.com/vue-bulma/datepicker/blob/master/src/WrapperInput.vue#L3

**Comment by fundon** [comment]
Try this:

```vue
            <datepicker :class="'test'" :value.sync="value"></datepicker>
```

**Comment by fundon** [comment]
Landed in v1.2.2

Try:

```vue
<datepicker :inputClass="{ test: true }"></datepicker>
```

**Comment by gyamxxx** [comment]
thanks! good works
